Automattic/wp-calypso
# Gutenlypso: Set the Classic block selected when clicked inside

#### Changes proposed in this Pull Request

* Add a `click` event binding to the Calypso `TinyMCE` component, and use it to select the Gutenberg Classic block whenever is clicked inside.

Tapping into the TinyMCE event API is required because otherwise we wouldn't be able to listen to a click event happening inside an iframe.

#### Testing instructions



* Open Gutenlypso at `/block-editor`.
* Insert a Classic block.
* Try clicking in and out the block (both on the toolbar and on the editor iframe), and make sure the block selection toggles as expected.
* Make sure the block toolbar (not the editor toolbar, which should be always visible) appears every time the block is selected.

Fixes #29569
